src/modules/pece/pece_security/pece_security.info

```diff
@@ -0,0 +1,37 @@
+name = PECE Security
+description = PECE security provisions
+core = 7.x
+package = Features
+version = 7.x-1.0-alpha1
+dependencies[] = aes
+dependencies[] = defaultconfig
+dependencies[] = features
+dependencies[] = login_security
+features[defaultconfig][] = strongarm:aes_cipher
+features[defaultconfig][] = strongarm:aes_encryption_iv
+features[defaultconfig][] = strongarm:aes_implementation
+features[defaultconfig][] = strongarm:aes_key
+features[defaultconfig][] = strongarm:aes_viewing_method
+features[defaultconfig][] = strongarm:login_security_activity_threshold
+features[defaultconfig][] = strongarm:login_security_disable_core_login_error
+features[defaultconfig][] = strongarm:login_security_host_hard_banned
+features[defaultconfig][] = strongarm:login_security_host_soft_banned
+features[defaultconfig][] = strongarm:login_security_host_wrong_count
+features[defaultconfig][] = strongarm:login_security_host_wrong_count_hard
+features[defaultconfig][] = strongarm:login_security_last_access_timestamp
+features[defaultconfig][] = strongarm:login_security_last_login_timestamp
+features[defaultconfig][] = strongarm:login_security_login_activity_email_body
+features[defaultconfig][] = strongarm:login_security_login_activity_email_subject
+features[defaultconfig][] = strongarm:login_security_login_activity_email_user
+features[defaultconfig][] = strongarm:login_security_notice_attempts_available
+features[defaultconfig][] = strongarm:login_security_notice_attempts_message
+features[defaultconfig][] = strongarm:login_security_track_time
+features[defaultconfig][] = strongarm:login_security_user_blocked
+features[defaultconfig][] = strongarm:login_security_user_blocked_email_body
+features[defaultconfig][] = strongarm:login_security_user_blocked_email_subject
+features[defaultconfig][] = strongarm:login_security_user_blocked_email_user
+features[defaultconfig][] = strongarm:login_security_user_wrong_count
+features[defaultconfig][] = user_default_permissions:administer aes
+features[features_api][] = api:2
+features[user_permission][] = administer aes
```

Comment:
This line should be removed in favor of the corresponding default config permission exported above (line 34).